L1Trigger/TrackFindingTracklet/src/MatchCalculator.cc

```diff
@@ -384,6 +415,21 @@ void MatchCalculator::execute(unsigned int iSector, double phioffset) {
         idrcut = rcut2Stable_.lookup(seedindex);
       }
 
+      unsigned int projindex = mergedMatches[j].first.second;  // Allproj index
+      curr_projid = next_projid;
+      next_projid = projindex;
+      // Do we have a new tracklet?
+      bool newtracklet = (j == 0 || projindex != curr_projid);
+      // If so, replace the "best" values with the cut tables
+      if (newtracklet) {
+        best_ideltaphi_disk = idrphicut;
+        best_ideltar_disk = idrcut;
+      }
+
+      // Update the cut vales (cut table if new tracklet, otherwise current best)
```

Comment:
Given the if block above, the following two lines are a tautology and must be removed.
Unless there is a flaw in the logic... please check

Reply:
@bryates I agree with @perrotta that lines 430-431 look weird. As you last touched this code, could you please check it?
e.g. The line " idrphicut = newtracklet ? idrphicut : best_ideltaphi_disk;" does absolutely nothing if newtracklet=true, so could be simplified to "if (not newtracklet) idrphicut = best_ideltaphi_disk.". But please check if it makes logical sense.

Reply:
Sorry for the delay. Yes, these lines is indeed unneeded, so I've removed them.